lua/astrocommunity/completion/avante-nvim/init.lua

```diff
@@ -78,5 +78,43 @@ return {
         opts.filetypes = require("astrocore").list_insert_unique(opts.filetypes, { "Avante" })
       end,
     },
+    {
+      "nvim-neo-tree/neo-tree.nvim",
+      optional = true,
+      opts = function(_, opts)
+        local ok, _ = pcall(require, "neo-tree")
+        if not ok then return opts end
+        return require("astrocore").extend_tbl(opts, {
```

Comment:
This should probably be rewritten to just use the opts syntax directly. See snacks-picker